### PR TITLE
New 1 tasks admin page layout havron

### PR DESCRIPTION
**Pull request by OleksandrHavron**

### CODE REVIEW

File: front-end/src/Components/pages/tasksmanagingPage/TaskCard.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+import CategoriesDropDown from '../../Common/CategoriesDropDown';
+import LanguagesDropDown from '../../Common/LanguagesDropDown';
+
+
+class TaskCard extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false,
+            task: props.task,
```
Comment (vvelizariy) [inline front-end/src/Components/pages/tasksmanagingPage/TaskCard.js]: Is there any sense to put task to state, if you doesn't modify it. Just use directly from props.

File: front-end/src/Components/pages/tasksmanagingPage/TaskManagingPage.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        }
+    }
+
+    async getTasks() {
+        const response = await axios.get('/task/get_task/');
+        return response.data;
+    }
+
+    async componentDidMount() {
+        this.getTasks().then(response => {
```
Comment (vvelizariy) [inline front-end/src/Components/pages/tasksmanagingPage/TaskManagingPage.js]: Try not no mix async with callbacks.

File: front-end/src/Components/pages/tasksmanagingPage/TaskManagingPage.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                            Created
+                        </div>
+                        <div className="task-shortinfo-updated inline">
+                            Updated
+                        </div>
+                    </div>
+                    {tasks.length === 0
+                        ? <h1>No tasks found</h1>
+                        : tasks.map(task =>
+                            <>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/tasksmanagingPage/TaskManagingPage.js]: Seems line wrapping into Fragment is not required there.

File: front-end/src/Components/pages/tasksmanagingPage/taskmanagingpage.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    width: 15%;
+}
+
+.task-shortinfo-created {
+    width: 15%;
+}
+
+.task-shortinfo-updated {
+    width: 15%;
+}
```
Comment (Viktorika-p) [inline front-end/src/Components/pages/tasksmanagingPage/taskmanagingpage.css]: Duplication shoud be avoid here.

Comment (OleksandrHavron) [reply front-end/src/Components/pages/tasksmanagingPage/taskmanagingpage.css]: How?

Comment (Viktorika-p) [reply front-end/src/Components/pages/tasksmanagingPage/taskmanagingpage.css]: Example:

.about_page, .teamtest {
    background-color: #292929;
}

try:
.task-shortinfo-created, .task-shortinfo-updated  {
    width: 15%;
}

**Review by vvelizariy — APPROVED** [review]

**Review by MetalMari — APPROVED** [review]

**Review by s0fko — APPROVED** [review]

**Pull request merged**